doc/usage.md

```diff
@@ -253,47 +253,55 @@ Please read upgrade notes if you're upgrading to a newer version.
+[2021-08-17T22:12:16.957Z INFO  electrs::index] indexing 2000 blocks: [692001..694000]
+[2021-08-17T22:13:11.764Z INFO  electrs::chain] chain updated: tip=00000000000000000003f5acb5ec81df7c98c16bc8d89bdaadd4e8965729c018, height=694000
+[2021-08-17T22:13:11.777Z INFO  electrs::index] indexing 2000 blocks: [694001..696000]
+[2021-08-17T22:14:05.852Z INFO  electrs::chain] chain updated: tip=0000000000000000000dfc81671ac5a22d8751f9c1506689d3eaceaef26470b9, height=696000
+[2021-08-17T22:14:05.855Z INFO  electrs::index] indexing 295 blocks: [696001..696295]
+[2021-08-17T22:14:15.557Z INFO  electrs::chain] chain updated: tip=0000000000000000000eceb67a01c81c65b538a7b3729f879c6c1e248bb6577a, height=696295
+[2021-08-17T22:14:21.578Z INFO  electrs::db] starting config compaction
+[2021-08-17T22:14:21.623Z INFO  electrs::db] starting headers compaction
+[2021-08-17T22:14:21.667Z INFO  electrs::db] starting txid compaction
+[2021-08-17T22:22:27.009Z INFO  electrs::db] starting funding compaction
+[2021-08-17T22:38:17.104Z INFO  electrs::db] starting spending compaction
+[2021-08-17T22:55:11.785Z INFO  electrs::db] finished full compaction
+[2021-08-17T22:55:15.835Z INFO  electrs::server] serving Electrum RPC on 127.0.0.1:50001
+[2021-08-17T22:55:25.837Z INFO  electrs::index] indexing 7 blocks: [696296..696302]
+[2021-08-17T22:55:26.120Z INFO  electrs::chain] chain updated: tip=0000000000000000000059e97dea0b0b9ebf4ac1fd66726b339fe1c9683de656, height=696302
+[2021-08-17T23:02:03.453Z INFO  electrs::index] indexing 1 blocks: [696303..696303]
+[2021-08-17T23:02:03.691Z INFO  electrs::chain] chain updated: tip=000000000000000000088107c337bf315e2db1e406c50566bd765f04a7e459b6, height=696303
 ```
 You can specify options via command-line parameters, environment variables or using config files.
 See the documentation above.
 
-Note that the final DB size should be ~20% of the `blk*.dat` files, but it may increase to ~35% at the end of the inital sync (just before the [full compaction is invoked](https://github.com/facebook/rocksdb/wiki/Manual-Compaction)).
+Note that the final DB size should be ~10% of the `blk*.dat` files, but it may increase to ~20% at the end of the inital sync (just before the [full compaction is invoked](https://github.com/facebook/rocksdb/wiki/Manual-Compaction)).
 
-If initial sync fails due to `memory allocation of xxxxxxxx bytes failedAborted` errors, as may happen on devices with limited RAM, try the following arguments when starting `electrs`.
-It should take roughly 18 hours to sync and compact the index on an ODROID-HC1 with 8 CPU cores @ 2GHz, 2GB RAM, and an SSD using the following command:
-
-```bash
-$ ./target/release/electrs -vvvv --index-batch-size=10 --jsonrpc-import --db-dir ./db --electrum-rpc-addr="127.0.0.1:50001"
-```
+It should take roughly 18 hours to sync and compact the index on an ODROID-HC1 with 8 CPU cores @ 2GHz, 2GB RAM, and an SSD using the command above.
```

Comment:
> Is this value still the same given that others changed? 

I think that it didn't change much - https://github.com/romanz/electrs/issues/308#issuecomment-905887940.

> Also I don't remember any change causing memory to no longer be an issue 

Previously, we were loading full `blk*.dat` files into RAM, and indexing them in a separate thread(s) - which resulted in high memory usage.

Now the indexing works block-by-block (received from bitcoind via p2p interface) without any concurrency - quite similar to JSONRPC-based sync in the previous version.
BTW, using p2p instead of JSONRPC also resolved https://github.com/romanz/electrs/issues/373 :)